clang/test/Preprocessor/_Pragma-in-include.c

```diff
@@ -0,0 +1,5 @@
+// RUN: %clang_cc1 %s -verify
```

Comment:
```suggestion
// RUN: %clang_cc1 -E %s -verify
```
This ensures we only run the preprocessing steps, which should still report the diagnostics.